numpy/core/src/umath/ufunc_object.c

```diff
@@ -864,6 +880,17 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
                         bad_arg = 0;
                     }
                     break;
+                case 'k':
+                    if (out_keepdims != NULL && strcmp(str, "keepdims") == 0) {
+                        if (!PyBool_Check(value)) {
+                            PyErr_SetString(PyExc_TypeError,
+                                        "'keepdims' must be a boolean");
+                            goto fail;
+                        }
+                        *out_keepdims = (value == Py_True);
```

Comment:
What if `value` is `np.false_`? How do we handle this in `subok`?

Reply:
I simply copied the stanza from `subok`, but agree it is somewhat odd. I'd prefer to leave it for a quick `MAINT` PR, though.